hoff.cabal

```diff
@@ -82,8 +80,11 @@ executable hoff
   default-language: GHC2021
   main-is:          Main.hs
   hs-source-dirs:   app
+
+  -- https://downloads.haskell.org/ghc/latest/docs/users_guide/runtime_control.html#rts-options-to-produce-runtime-statistics
   ghc-options:
     -Wall -Werror -Wincomplete-uni-patterns -Wincomplete-record-updates
+    -with-rtsopts=-T
```

Comment:
This looks useful, but why is it added in this PR? Does it make the metrics better?